pages/blog/release-notes-2.1.0.md

```diff
@@ -39,4 +41,11 @@ Thanks to [Dale Lane](https://github.com/dalelane) you can now describe secured
 
 We used and supported `defaultContentType` property to specify default content type to use when encoding/decoding a message's payload. [Lucas Block](https://github.com/BlockLucas) spotted that we do not have it defined in the specification. For more details check out [this pull request](https://github.com/asyncapi/spec/pull/419).
 
+## Tooling support
+
+The following tools are already updated to support 2.1.0 version of the specification:
+- JSON Schema that supports validation of AsyncAPI documents is updated in [this](https://github.com/asyncapi/asyncapi-node) repository. Also `@asyncapi/specs` package has been updated on NPM to version 2.8.0 and it contains the 2.1.0 JSON Schema.
+- [JavaScript Parser](https://github.com/asyncapi/parser-js/) uses latest `@asyncapi/specs` package and can be used to parse and validate `2.1.0` documents. Upgrade to 1.6.0 version.
```

Comment:
Remember to change `1.6.0` version to the latest.

Reply:
yup, will keep that one unresolved